lensless/unrolled_admm.py

```diff
@@ -0,0 +1,248 @@
+# #############################################################################
+# unrolled_admm.py
+# =================
+# Authors :
+# Yohann PERRON [yohann.perron@gmail.com]
+# #############################################################################
+
+import numpy as np
+from lensless.trainable_recon import TrainableReconstructionAlgorithm
+from lensless.admm import soft_thresh, finite_diff, finite_diff_adj, finite_diff_gram
+from scipy import fft
+
+try:
+    import torch
+
+    torch_available = True
+except ImportError:
+    torch_available = False
```

Comment:
do we need this variable in this file? I guess `torch` is needed and you can do a same error message as in `UnrolledFISTA`